torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2399,6 +2399,14 @@ def error_inputs_masked_select(op_info, device, **kwargs):
                      error_type=RuntimeError,
                      error_regex='unsupported operation')
 
+def error_inputs_median(op_info, device, **kwargs):
+    x = torch.tensor([[[[[[[[[[[[[[[[[[[[[[[[[nan],
+                               [nan]]]]]]]]]]]]]]]]]]]]]]]]], device=device)
+    yield ErrorInput(SampleInput(x, kwargs=dict(dim=(-1))),
```

Comment:
you should yield error input only if `device.type=="cuda"`, otherwise you should just return (this input won't produce error on CPU).